indico/modules/events/papers/templates/display/_common.html

```diff
@@ -0,0 +1,38 @@
+{% macro render_cfp_infoline(event, cfp, show_icon=true, redirect_first=false) %}
+    {% set can_submit_papers = cfp.is_open %}
+    <div class="infoline announcement">
+        {% if show_icon %}
+            <span class="icon icon-file-content"></span>
+        {% endif %}
+        <div class="text">
+            {%- if can_submit_papers %}
+                <div class="label">
+                    {%- trans %}The call for papers is open{% endtrans -%}
+                </div>
+                <div>
+                    {%- trans %}You can submit a paper for reviewing.{% endtrans -%}
+                </div>
+                <div class="toolbar right">
+                    {% if session.user and not redirect_first %}
+                        <a class="i-button highlight js-show-paper-form"
+                           data-title="{% trans %}Submit new paper{% endtrans %}"
+{#                           TODO: Link to submit paper #}
+{#                           data-href="{{ url_for('papers.submit', event) }}"#}
```

Comment:
Then maybe this should actually show a dropdown where you can select the contribution in case you have more than one (and point to the submission page for your contribution directly if you have only one)?